lms/djangoapps/learner_home/test_serializers.py

```diff
@@ -835,7 +835,7 @@ def create_test_data(cls, enrollment):
     def create_test_context(cls, enrollment):
         """Credit data, packaged as it would be for serialization context"""
 
-        return {enrollment.course.id: {**cls.create_test_data(enrollment)}}
+        return {enrollment.course_id: {**cls.create_test_data(enrollment)}}
```

Comment:
Not strictly necessary, but we usually use `course_id` instead of `course.id` so bringing this into alignment